aries-site/src/pages/templates/forms.mdx

```diff
@@ -250,3 +258,40 @@ In cases where the user is given an opportunity to express themselves or would l
   href="#sign-in"
   alt="HPE Design System Form where required and optional can be skipped"
 />
+
+## Full-page form
+
+- Use full-page forms for create and edit flows, irrespective of the size of the form.
```

Comment:
What if I am creating or editing an object with only two fields?

I can't select an object from a list and edit its attributes in a side-layer?